modules/perform-on-demand-backups-s3-api-compatible-services.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * integration/integrate-with-s3-api-compatible-services.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="perform-on-demand-backups-s3-api-compatible-services_{context}"]
+= Performing on-demand backups on an S3 API compatible bucket
+
+Use the {product-title-short} portal to trigger manual backups of {product-title} to an S3 API compatible bucket.
+
+.Prerequisites
+* You must have already integrated {product-title} with an S3 API compatible bucket.
+
+.Procedure
+. In the {product-title-short} portal, go to *Platform Configuration* -> *Integrations*.
+. Under the *External backups* section, click *S3 API Compatible*.
+. Select the integration name for the S3 bucket where you want to do a backup.
+. Click *Trigger Backup*.
```

Comment:
Is backup capitalized? (I think Patternfly style has sentence case so it shouldn't be, I think?)

Reply:
You are right, backup is not capitalized. I'll change it.